src/views/Biobanker.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <navigation-bar />
+    <div class="container">
+        <NegotiationList :negotiations="negotiations"/>
+    </div>
+</template>
+
+<script>
+import NegotiationList from "@/components/NegotiationList.vue";
+import NavigationBar from "@/components/NavigationBar.vue";
+import {mapActions, mapGetters} from "vuex";
+
+export default {
+    name: "Biobanker",
```

Comment:
usually, names are lowercase. Then you can use both <Biobanker/> from the file name and <biobanker> from the name